### ISSUE TITLE
`no-manually-wrapped-comments`: allow comments prefixed with an all-caps tag

### ISSUE THREAD

**Issue by sergiocarneiro** [sindresorhus/eslint-plugin-unicorn#3459]
The rule wants to merge these two comments into one:

```js
// MARK: - Networking
// Handles requests, retries, and caching
```

In this case, the comments are related, but they are not a wrapped sentence. The first line is a standalone section marker, and the second line is its caption or description.

This also comes up with stacked tagged comments such as `TODO:`, `FIXME:`, `NOTE:`, and similar annotations, where each line is intended to remain distinct.

Adding punctuation to the tag line avoids the warning:

```js
// MARK: - Networking.
```

But that reads awkwardly for a section header.

Could the rule ignore comments that begin with an all-caps tag? These read as standalone annotations rather than wrapped prose.
